### DISCUSSION TITLE
Where in the code is the feature where a right click actually looks like Windows 11?

### DISCUSSION THREAD

**Discussion by ask-arim** [blueedgetechno/win11React#705] [Q&A]
Where in the code is the feature where a right click actually looks like Windows 11?

**Comment by Wrinzon** [comment]
Should I answer here or in the website？



At 2022-12-05 20:58:44, "ask-arim" ***@***.***> wrote:

Where in the code is the feature where a right click actually looks like Windows 11?

**Comment by Wrinzon** [comment]
And sorry,I'm not very good at English.Can you reply me in Chinese?



At 2022-12-05 20:58:44, "ask-arim" ***@***.***> wrote:

Where in the code is the feature where a right click actually looks like Windows 11?

**Comment by Wrinzon** [comment]
Sorry I don't know what is that meaning.I said I only know a few of English.Please translate.



At 2022-12-09 13:00:40, "Wrinzon" ***@***.***> wrote:

And sorry,I'm not very good at English.Can you reply me in Chinese?



At 2022-12-05 20:58:44, "ask-arim" ***@***.***> wrote:

Where in the code is the feature where a right click actually looks like Windows 11?

**Comment by Wrinzon** [comment]
这什么傻逼玩意，怎么总是重复我发的信息



At 2022-12-09 13:03:09, "Wrinzon" ***@***.***> wrote:

Sorry I don't know what is that meaning.I said I only know a few of English.Please translate.



At 2022-12-09 13:00:40, "Wrinzon" ***@***.***> wrote:

And sorry,I'm not very good at English.Can you reply me in Chinese?



At 2022-12-05 20:58:44, "ask-arim" ***@***.***> wrote:

Where in the code is the feature where a right click actually looks like Windows 11?

**Comment by gabrieljemail** [comment]
Maybe don't email replys
